pkg/cmd/experimental/syncgroups/schema3/config.go

```diff
@@ -0,0 +1,28 @@
+package schema3
```

Comment:
Move this entire package out of this pull.  I'm not going to review it until after schema1 is done.